ggplot_base_r_graph_analysis.Rmd

```diff
@@ -0,0 +1,115 @@
+---
```

Comment:
The whole YAML introduction (the part with title: author: etc...) should be cancelled including the --- lines . Instead, fix it with an introduction like this:
#Title of your project
_white line_
Your name
_white line _

Put the hashtag in front of the title of your project as it is marked above.